### ISSUE TITLE
Crypto polyfill always uses `import`

### ISSUE THREAD

**Issue by gsvarovsky** [belayeng/quadstore-comunica#2]
https://github.com/belayeng/quadstore-comunica/blob/20177a38f1790a33828684702f3655f1fdb2a8ea/engine/src/crypto-polyfill.js#L23

This check doesn't work because the Function constructor uses global scope for executing its parameter, but `require` is available in module scope.

In a debugger:
```
→ Function('return typeof require === \'function\'')()
false
→ typeof require === 'function'
true
→ global.require
undefined
```

This causes a problem with Jest, which injects its own module loaders, and import fails.

Is it possible to fool webpack using a variable? That's what I do for browserify, e.g.:

```
  if (typeof require === 'function') { // Probably safe, no?

    const cryptoModule = 'crypto'; // Does this fool webpack?
    polyfillCrypto(require(cryptoModule));

  }
```

**Comment by jacoscaz** [comment]
Hey @gsvarovsky ! The issue there would be the if condition `(typeof require === 'function')`, which Webpack somehow translates to `true`. Previous hack was to use `eval` but I wasn’t too happy with it. I can go back to it if nothing else works , though.

**Comment by jacoscaz** [comment]
@gsvarovsky could you try with version `3.0.3-beta.1`, which I've just published to NPM?

**Comment by gsvarovsky** [comment]
It works! 💯

**Comment by jacoscaz** [comment]
Fixed in version `3.0.3` just published to NPM.
